### PR TITLE
[v6r15] Added default methods to the FileCatalogClientBase

### PR DESCRIPTION
**Pull request by atsareg**
Added default methods needed to be implemented by all the catalogs to be used in standard operations:

hasAccess()
exists()
getPathPermissions

In the default implementations the answer is True to all the requests

### CODE REVIEW

**Comment by chaen** [comment]
I think this implementation using class attributes is much better. 
However returning True by default for all permission checking is a bit dangerous no ?

File: Resources/Catalog/FileCatalogClientBase.py [hunk 1]
```diff
@@ -35,17 +54,65 @@ def isOK( self, timeout = 120 ):
...
+      :param lfn paths: has to be formatted this way :
+                  { lfn : { se1 : pfn1, se2 : pfn2, ...}, ...}
+
+      :return: { successful : { lfn : [ ses ] } : failed : { lfn : { se : msg } } }
+
+    """
+    return S_OK( {'Failed' : {}, 'Successful' : dict.fromkeys( lfns, False )} )
+
+  @checkCatalogArguments
+  def getPathPermissions( self, lfns ):
```
Comment (chaen) [inline Resources/Catalog/FileCatalogClientBase.py]: Why not doing it the same way are the two others using fromkeys ?

**Comment by atsareg** [comment]
@chaen : The default implementation to return True for all the permissions is for the pseudocatalogs where no permissions are assumed at all. But for those path filters will apply that you have developed

**Comment by atsareg** [comment]
I have a strange glitch in the commit comments in my PyCharm... spotted, will try to fix

**Comment by chaen** [comment]
yes I understand that, I was just thinking of new catalogs. But well, that does not happen everyday, so review ok

**Pull request merged**